CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 2.0.2
```

Comment:
The version in the pubspec should always match the latest version in the changelog (or have the `-dev` suffix if you're not planning to release that fix immediately)

Reply:
The version of the pubspec is matched to `2.0.2`.. am I missing something?

Reply:
Oh, whoops. My bad. Somehow missed it